luatest/server_v2_instance.lua

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env tarantool
+
+local fio = require('fio')
+local fun = require('fun')
+local json = require('json')
+
+local TIMEOUT_INFINITY = 500 * 365 * 86400
+
+local function default_cfg()
+    return {
+        work_dir = os.getenv('TARANTOOL_WORKDIR'),
+        listen = os.getenv('TARANTOOL_LISTEN'),
+        log = fio.pathjoin(
+            os.getenv('TARANTOOL_WORKDIR'),
+            os.getenv('TARANTOOL_ALIAS') .. '.log'
+        ),
+    }
+end
+
+local function env_cfg()
+    local cfg = os.getenv('TARANTOOL_BOX_CFG')
+    if cfg == nil then
+        return {}
+    end
+    local res = json.decode(cfg)
+    assert(type(res) == 'table')
+    return res
+end
+
+local function box_cfg(cfg)
+    return fun.chain(default_cfg(), env_cfg(), cfg or {}):tomap()
+end
+
+-- Set the shutdown timeout to infinity so that we can catch tests that leave
+-- asynchronous requests. If we used the default timeout of 3 seconds, such a
+-- test would still pass, but it would slow down the overall test run, because
+-- the server would take longer to stop. Setting the timeout to infinity makes
+-- such bad tests hang and fail.
```

Comment:
```suggestion
-- Set the shutdown timeout to infinity to catch tests that leave
-- asynchronous requests. With the default timeout of 3 seconds, such
-- test would still pass, but slow down the overall test run, because
-- the server would take longer to stop. Setting the timeout to infinity makes
-- such bad tests hang and fail.
```

Reply:
Fixed.